packages/openapi3/test/test-host.ts

```diff
@@ -77,10 +78,43 @@ export async function emitOpenApiWithDiagnostics(
   });
   const content = runner.fs.get(outputFile);
   ok(content, "Expected to have found openapi output");
-  const doc = fileType === "json" ? JSON.parse(content) : parse(content);
+  const doc = fileType === "json" ? JSON.parse(content) : yamlParse(content);
   return [doc, diagnostics, content];
 }
 
+export async function emitOpenApi(
```

Comment:
this feels like we already have all the helpers needed for this, why is this added? the signature also looks bad a bunch of callbacks with no names

Reply:
Could you share which helper I can use? looks like none of them return fs. Or I added fs to one of the helpers? Or I use `createOpenAPITestRunner` directly?
I follow the helper `emitOpenApiWithDiagnostics`'s return `[doc, diagnostics, content]` which is already looks bad, I can make the return a dict to make it clearer.

Reply:
check output-file.test.ts where this test should be anyway